ndonnx/_core/_numericimpl.py

```diff
@@ -357,14 +291,14 @@ def argmax(self, x, axis=None, keepdims=False):
         if axis is None:
             reshaped_x = ndx.reshape(x, [-1])._core()
             if keepdims:
-                return _from_corearray(
+                return from_corearray(
```

Comment:
Similar to my previous comment, I'm can't quite see why we are taking the detour via `_CoreArray` and `opx`. Why not directly call into the Spox operators by default?

Reply:
As you mentioned there, we're doing eager propagation at that level of the call stack. I'd prefer to move broader discussion about this to its own GitHub Issue/offline if that's okay.